src/pages/offer-page/offer-page.tsx

```diff
@@ -17,130 +30,110 @@ export default function OfferPage(): JSX.Element {
-                    Fridge
-                  </li>
-                </ul>
-              </div>
+              {offerDetails.goods.length > 0 && (
+                <div className="offer__inside">
+                  <h2 className="offer__inside-title">What&apos;s inside</h2>
+                  <ul className="offer__inside-list">
+                    {offerDetails.goods.map((item) => (
+                      <li className="offer__inside-item" key={item}>
+                        {item}
+                      </li>
+                    ))}
+                  </ul>
+                </div>
+              )}
               <div className="offer__host">
                 <h2 className="offer__host-title">Meet the host</h2>
                 <div className="offer__host-user user">
-                  <div className="offer__avatar-wrapper offer__avatar-wrapper--pro user__avatar-wrapper">
-                    <img className="offer__avatar user__avatar" src="img/avatar-angelina.jpg" width="74" height="74" alt="Host avatar" />
+                  <div className={cn(
+                    'offer__avatar-wrapper',
+                    { 'offer__avatar-wrapper--pro': offerDetails.host.isPro },
+                    'user__avatar-wrapper'
+                  )}
+                  >
+                    <img className="offer__avatar user__avatar" src={offerDetails.host.avatarUrl} width="74" height="74" alt="Host avatar" />
                   </div>
                   <span className="offer__user-name">
-                    Angelina
+                    {offerDetails.host.name}
                   </span>
-                  <span className="offer__user-status">
+                  {offerDetails.host.isPro && (
+                    <span className="offer__user-status">
                     Pro
-                  </span>
+                    </span>
+                  )}
                 </div>
                 <div className="offer__description">
                   <p className="offer__text">
-                    A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century.
-                  </p>
-                  <p className="offer__text">
-                    An independent House, strategically located between Rembrand Square and National Opera, but where the bustle of the city comes to rest in this alley flowery and colorful.
+                    {offerDetails.description}
                   </p>
                 </div>
               </div>
               <section className="offer__reviews reviews">
-                <h2 className="reviews__title">Reviews &middot; <span className="reviews__amount">1</span></h2>
-                <ReviewList />
-                <NewReview />
+                <h2 className="reviews__title">Reviews &middot; <span className="reviews__amount">{commentsList.length}</span></h2>
+                {commentsList.length > 0 && (
```

Comment:
```suggestion
                {commentsList.length > 0 && (
```

Проверку `commentsList.length > 0 ` можно перенести внутрь компонента